README.md

```diff
@@ -6,149 +6,96 @@
 
 This repository contains [Ansible](https://www.ansible.com/) roles and
 playbooks to install, upgrade, and manage
-[OpenShift](https://www.openshift.com/) clusters.
+[OpenShift](https://www.openshift.com/) clusters on RHEL hosts.
 
-**Note**: the Ansible playbooks in this repository require an RPM
-package that provides `docker`. Currently, the RPMs from
-[dockerproject.org](https://dockerproject.org/) do not provide this
-requirement, though they may in the future. This limitation is being
-tracked by
-[#2720](https://github.com/openshift/openshift-ansible/issues/2720).
+## Getting Started
 
-## Getting the correct version
-When choosing an openshift release, ensure that the necessary origin packages
-are available in your distribution's repository.  By default, openshift-ansible
-will not configure extra repositories for testing or staging packages for
-end users.
-
-We recommend using a release branch. We maintain stable branches
-corresponding to upstream Origin releases, e.g.: we guarantee an
-openshift-ansible 3.2 release will fully support an origin
-[1.2 release](https://github.com/openshift/openshift-ansible/tree/release-1.2).
-
-The most recent branch will often receive minor feature backports and
-fixes. Older branches will receive only critical fixes.
-
-In addition to the release branches, the master branch
-[master branch](https://github.com/openshift/openshift-ansible/tree/master)
-tracks our current work **in development** and should be compatible
-with the
-[Origin master branch](https://github.com/openshift/origin/tree/master)
-(code in development).
+### Requirements
 
+#### Ansible
+[Ansible 2.7](https://docs.ansible.com/ansible/2.5/installation_guide/intro_installation.html) 
 
+#### Packages
+```
+pip install -U apache-libcloud~=2.2.1
+pip install google-auth
+yum install -y google-cloud-sdk python2-crypto
+```
 
-**Getting the right openshift-ansible release**
+#### GCP Credentials
 
-Follow this release pattern and you can't go wrong:
+Place your service account key with the name `gce.json` in the root of OpenShift-Ansible.
 
-| Origin/OCP    | OpenShift-Ansible version | openshift-ansible branch |
-| ------------- | ----------------- |----------------------------------|
-| 1.3 / 3.3          | 3.3               | release-1.3 |
-| 1.4 / 3.4          | 3.4               | release-1.4 |
-| 1.5 / 3.5          | 3.5               | release-1.5 |
-| 3.*X*         | 3.*X*             | release-3.x |
+OpenShift developers can find the service account key in the [shared secrets repo](https://github.com/openshift/shared-secrets/blob/master/gce/aos-serviceaccount.json).
```

Comment:
Generating a service account key is straight forward. I decided not to link to the docs for generating a service account key because OpenShift team members don't need to generate new ones. I removed the links to the private repo.